Doc/whatsnew/3.11.rst

```diff
@@ -739,11 +745,34 @@ For major changes, see :ref:`new-feat-related-type-hints-311`.
   to clear all registered overloads of a function.
   (Contributed by Jelle Zijlstra in :gh:`89263`.)
 
-* :data:`typing.TypedDict` subclasses can now be generic. (Contributed by
-  Samodya Abey in :gh:`89026`.)
+* The :meth:`__init__` method of :class:`~typing.Protocol` subclasses
+  is now preserved. (Contributed by Adrian Garcia Badarasco in :gh:`88970`.)
 
-* :class:`~typing.NamedTuple` subclasses can now be generic.
-  (Contributed by Serhiy Storchaka in :issue:`43923`.)
+* The representation of empty tuple types (``Tuple[()]``) is simplified.
+  (Contributed by Serhiy Storchaka in :gh:`91137`.)
```

Comment:
After seeing your comment https://github.com/python/cpython/issues/91137#issuecomment-1093946369, I can see why you're including this, but this note feels a bit opaque at the moment (I didn't understand what it meant until I looked up the GitHub issue). What impact does this have for end users?

Reply:
How about:

```suggestion
* The representation of empty tuple types (``Tuple[()]``) is simplified.
  This affects introspection, e.g. ``get_args(Tuple[()])`` now evaluates to ``()`` instead of ``((),)``.
  (Contributed by Serhiy Storchaka in :gh:`91137`.)
```